include/openPMD/IO/AbstractIOHandler.hpp

```diff
@@ -61,7 +61,11 @@ class unsupported_data_error : public std::runtime_error
  * @brief Determine what items should be flushed upon Series::flush()
  *
  */
+#ifdef __NVCOMPILER_MAJOR__
```

Comment:
Is it worth adding this work-around instead of generally using `enum class` without a type?